tests/integration/components/frost-list-item-selection-test.js

```diff
@@ -27,7 +27,8 @@ describe(test.label, function () {
     this.on('selectAction', selectSpy)
     this.setProperties({
       hook: 'myListItemSelection',
-      model: model
+      model: model,
+      singleSelection: false
```

Comment:
With the specific setting of this property I am not seeing any test cases that are also ensuring that this value/behavior is `false` by default.  We should be testing the setting of it to `false` and `true` as is currently being done but ensuring that it is defaults to what is expected should be tested as well.

Normally I would approach that via a unit test but it can be done in an integration test as well.

Reply:
This comment still applies

Reply:
I added a unit test [here](https://github.com/ciena-frost/ember-frost-list/pull/186/commits/f9ce16d41622344d8d83952ae1afecbdd257e7b1#diff-371676dadeef8670e5698ffdac23445c) that checks if `singleSelection` is `false` when it is not provided. Are you looking to see integration tests without it set as well?

Reply:
Ahh, sorry missed that.  That is fine